contrib/cmark/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "cmark"
+pkgver = "0.31.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DBUILD_SHARED_LIBS=ON"]
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+]
+checkdepends = ["python"]
+pkgdesc = "C implementation of the CommonMark markdown specification"
+maintainer = "psykose <alice@ayaya.dev>"
+license = "BSD-2-Clause"
+url = "https://github.com/commonmark/cmark"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "bbcb8f8c03b5af33fcfcf11a74e9499f20a9043200b8552f78a6e8ba76e04d11"
+# defaults to Release which sets this, and the tests crash in an assert without it..
```

Comment:
and the assertion crash isn't a regular test failure that NDEBUG hides?

Reply:
technically, it is an actual assert (`src/blocks.c:262`), in api_test. but upstream never sets a build_type in their ci, so their own tests always run with ndebug from =release, so i bet it's actually broken for them too. a bit too lazy to ask however

Reply:
well ok, i guess that's fine

Reply:
actually i was wrong and `if(NOT CMAKE_BUILD_TYPE)` is false when `-DCMAKE_BUILD_TYPE=`(empty) which they pass, so they do have asserts enabled. will see what the issue is..

Reply:
durr, no. that's wrong, it indeed makes it Release for empty. i was just looking at something else

Reply:
so yeah, it's ok, and i'll open an issue anyway